test/system/components/student/solution_summary/mentoring_button_test.rb

```diff
@@ -46,6 +46,47 @@ class SolutionSummarySectionTest < ApplicationSystemTestCase
           assert_text "Want to get mentored by a friend?"
         end
       end
+
+      test "shows discussions within nudge section" do
```

Comment:
Are we showing the discussions or just a link to the mentoring? Maybe something like:

```suggestion
      test "shows links to continue mentoring within nudge section" do
```